spec/rubocop/cop/bundler/gem_version_spec.rb

```diff
@@ -24,6 +26,9 @@
         gem 'rubocop', '~> 1'
         gem 'rubocop', '~> 1.12', require: false
         gem 'rubocop', '>= 1.5.0', '< 1.10.0', git: 'https://github.com/rubocop/rubocop'
+        gem 'rubocop', github: 'rubocop/rubocop', tag: 'v1'
+        gem 'rubocop', git: 'https://github.com/rubocop/rubocop', tag: 'v1'
+        gem 'rubocop', bitbucket: 'https://bitbucket.com/foo/bar', tag: 'v1'
```

Comment:
`:ref` can be accepted as well as `:tag`.

Reply:
changed